index.html

```diff
@@ -1,79 +1,226 @@
           </div>
-          <div class="humberger menu"></div>
-          <div class="humberger">
-            <img src="humberger.svg" alt="humberger icon" />
+          <div class="card_works">
+           <img class="card_works_image" src="./assets/multi_post_1.svg" alt="image">
+           <h2 class="card_works_heading">Multi-Post Stories</h2>
+           <div class="card_works_links">
+             <span>Canopy</span>
+             &middot;
+             <span>Backend Dev</span>               
+             &middot;
+             <span>2015</span>
           </div>
-        </nav>
-      </header>
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <div class="card_works_tags">
+              <span>html</span>
+              <span>css</span>
+              <span>javascript</span>
+            </div>
+            <button class="card_works_button">See Project</button>
+         </div>
+         <div class="card_works">
+           <img class="card_works_image" src="./assets/tonic_2.svg" alt="image">
+           <h2 class="card_works_heading">Tonic</h2>
+           <div class="card_works_links">
+             <span>Canopy</span>
+             &middot;
+             <span>Backend Dev</span>               
+             &middot;
+             <span>2015</span>
+          </div>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <div class="card_works_tags">
+              <span>html</span>
+              <span>css</span>
+              <span>javascript</span>
+            </div>
+            <button class="card_works_button">See Project</button>
+         </div>
+         <div class="card_works">
+           <img class="card_works_image" src="./assets/multi_post_2.svg" alt="image">
+           <h2 class="card_works_heading">Multi-Post Stories</h2>
+           <div class="card_works_links">
+             <span>Canopy</span>
+             &middot;
+             <span>Backend Dev</span>               
+             &middot;
+             <span>2015</span>
+          </div>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <div class="card_works_tags">
+              <span>html</span>
+              <span>css</span>
+              <span>javascript</span>
+            </div>
+            <button class="card_works_button">See Project</button>
+         </div>
+        </section>
+
+        <div class="hero">
+          <section class="about flex">
+          <h1 class="about__title">
+            About <span>Myself</span>
+         </h1>
+         <p class="about__dsc">
+            Hello I'm a software developer! I can help you build a product , feature or
+            website Look through some of my work and experience! If you like what
+            you see and have a project you need coded, don’t hestiate to contact
+            me.
+         </p>
```

Comment:
Please ensure that you stick to the design as much as possible (e.g., font, colors, images, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). Please ensure that the `paragraph` has 6 lines as in the design.

Your Implementation | Figma Design
:---:|:---:
![image](https://user-images.githubusercontent.com/10332499/154817414-715c798b-77c2-4056-baf5-0cf9f232daa6.png) | ![image](https://user-images.githubusercontent.com/10332499/154817484-718d26d8-9310-4363-a8b9-89051af78153.png)